samples/msal-react-samples/nextjs-sample/src/ui-components/SignInButton.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+import { useMsal } from "@azure/msal-react";
+import Button from "@mui/material/Button";
+import MenuItem from '@mui/material/MenuItem';
+import Menu from '@mui/material/Menu';
+import { loginRequest } from "../authConfig";
+
+export const SignInButton = () => {
+    const { instance } = useMsal();
+
+    const [anchorEl, setAnchorEl] = useState(null);
+    const open = Boolean(anchorEl);
+
+    const handleLogin = (loginType) => {
```

Comment:
should this be async?

Reply:
doesn't matter since it doesn't return anything